frontend/src/components/Answer/Answer.tsx

```diff
@@ -323,9 +324,29 @@ export const Answer = ({ answer, onCitationClicked }: Props) => {
               </Stack>
             </Stack.Item>
           )}
-          <Stack.Item className={styles.answerDisclaimerContainer}>
-            <span className={styles.answerDisclaimer}>AI-generated content may be incorrect</span>
```

Comment:
Was this moved somewhere else?  Just want to be sure we still have the disclaimer somewhere.